pkg/bootstrap/docker/openshift/metrics.go

```diff
@@ -2,12 +2,11 @@ package openshift
 
 import (
 	"fmt"
-
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
 	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
 	kapi "k8s.io/kubernetes/pkg/api"
 	apierrors "k8s.io/kubernetes/pkg/api/errors"
-
-	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	kbatch "k8s.io/kubernetes/pkg/apis/batch"
```

Comment:
see if you can turn off the editor automatically rearranging the imports. 

We really should have something like:
```
	"fmt"

	kapi "k8s.io/kubernetes/pkg/api"
	apierrors "k8s.io/kubernetes/pkg/api/errors"
	kbatch "k8s.io/kubernetes/pkg/apis/batch"

	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
```